src/mc_tasks/ImpedanceTask.cpp

```diff
@@ -204,6 +225,10 @@ void ImpedanceTask::addToLogger(mc_rtc::Logger & logger)
   MC_RTC_LOG_HELPER(name_ + "_measuredWrench", measuredWrench_);
   MC_RTC_LOG_HELPER(name_ + "_filteredMeasuredWrench", filteredMeasuredWrench_);
   logger.addLogEntry(name_ + "_cutoffPeriod", this, [this]() { return cutoffPeriod(); });
+
+  // hold
```

Comment:
I don't think this comment is required

Reply:
Fixed in https://github.com/jrl-umi3218/mc_rtc/pull/143/commits/51d1eb078d2d27fc83085dd1e48545344e8face7